operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -204,6 +205,7 @@ private String getLockName(String namespace, String name) {
                     // and might not be really deleted. We have to filter these situations out and ignore the
                     // reconciliation because such resource might be already operated by another instance (where the
                     // same change triggered ADDED event).
+
```

Comment:
I do not think there is any reason to remove and add empty lines at random places - especially since you are not changing them at all. This comment is directly related to the log message below so the empty line is IMHO not needed.